### ISSUE TITLE
Policy output is interleaved with other output

### ISSUE THREAD

**Issue by joeduffy** [pulumi/pulumi-policy#167]
Notice the "3 unchanged" that comes after "Policy Packs run," not "Resources":

![image](https://user-images.githubusercontent.com/3953235/70018367-b9efac00-153a-11ea-97c6-d8ec84b1262b.png)

**Comment by justinvp** [comment]
I don't think this is still an issue. If it is, we should re-open in `pulumi/pulumi`.
